io.gdevs.GDLauncher.yml

```
@@ -1,52 +1,55 @@
+  - --socket=x11
+  - --share=ipc
+  - --device=all   # controller support
+  - --socket=pulseaudio
+  - --share=network
+  - --env=CHROME_WRAPPER=zypak-wrapper
+  - --filesystem=xdg-download   # downloaded/export resource packs etc.
 modules:
-    - name: openjdk
-      buildsystem: simple
-      build-commands:
-        - /usr/lib/sdk/openjdk/install.sh
-    - name: gdlauncher
-      buildsystem: simple
-      build-commands:
-        - chmod +x GDLauncher-linux-setup.AppImage
-        - ./GDLauncher-linux-setup.AppImage --appimage-extract
-        - cp -r squashfs-root /app/bin/gdlauncher
-        - install -D gdlauncherscaled.png /app/share/icons/hicolor/512x512/apps/io.gdevs.GDLauncher.png
-        - install -D io.gdevs.GDLauncher.desktop /app/share/applications/io.gdevs.GDLauncher.desktop
-        - install -D io.gdevs.GDLauncher.metainfo.xml /app/share/metainfo/io.gdevs.GDLauncher.appdata.xml
-        - install -D gd.sh /app/bin
-      sources:
-          - type: file
-            url: https://github.com/gorilla-devs/GDLauncher/releases/latest/download/GDLauncher-linux-setup.AppImage
-            sha256: 83192422964903efe4a9a84b995861f506a9f47252bb8cc45f5fdd99939ba09a
-            x-checker-data:
-              type: rotating-url
-              url: https://github.com/gorilla-devs/GDLauncher/releases/latest/download/GDLauncher-linux-setup.AppImage
-          - type: file
-            path: io.gdevs.GDLauncher.desktop
-          - type: file
-            path: gdlauncherscaled.png
-          - type: file
-            path: io.gdevs.GDLauncher.metainfo.xml
-          - type: script
-            commands:
-              - zypak-wrapper /app/bin/gdlauncher/AppRun
-            dest-filename: gd.sh
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk/install.sh
+  - name: gdlauncher
+    buildsystem: simple
+    build-commands:
+      - chmod +x GDLauncher-linux-setup.AppImage
+      - ./GDLauncher-linux-setup.AppImage --appimage-extract
+      - cp -r squashfs-root /app/bin/gdlauncher
+      - install -D gdlauncherscaled.png /app/share/icons/hicolor/512x512/apps/io.gdevs.GDLauncher.png
+      - install -D io.gdevs.GDLauncher.desktop /app/share/applications/io.gdevs.GDLauncher.desktop
+      - install -D io.gdevs.GDLauncher.metainfo.xml /app/share/metainfo/io.gdevs.GDLauncher.appdata.xml
+      - install -D gd.sh /app/bin
+    sources:
+      - type: file
+        url: https://github.com/gorilla-devs/GDLauncher/releases/download/v1.1.10/GDLauncher-linux-setup.AppImage
+        sha256: 83192422964903efe4a9a84b995861f506a9f47252bb8cc45f5fdd99939ba09a
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/gorilla-devs/GDLauncher/releases/latest
+          version-query: .tag_name | sub("^[vV]"; "")
+          url-query: .assets[] | select(.name=="GDLauncher-linux-setup.AppImage")
+            | .browser_download_url
```

To keep it on single line,
```suggestion
          url-query: |
            .assets[] | select(.name=="GDLauncher-linux-setup.AppImage") | .browser_download_url
```